caom2repo/caom2repo/core.py

```diff
@@ -471,6 +470,7 @@ def get_observation(self, collection, observation_id):
         """
         assert collection is not None
         assert observation_id is not None
```

Comment:
`observation_id` vs `uri`

which one is the better parameter?

Reply:
`observation_id` is confusing but I'm not sure whether it will be replaced with `uri` or something else. So this is technical debt.